index.html

```diff
@@ -0,0 +1,166 @@
+                        a aprheh saivia u wiainuvbuhr s olbuhids urola urhu ubrbu aouh vuauhruh;
+                        ajpaonplav ovbola;lvo rbvyoou fuvbr afubaou huahru ar vhaluh fuahropairuoi 
+                        avuwborlandubawoub dsauvuabwvboaubv uabvuibwyrguaiih 
+                        auwosurhao ousfafgu  ybbiy hbbiviyv kbjo oiub dob oabofy bcakbf asvbbasbal
+                        kslbvosuab fkb sajbdvusufukjasdb fb asdb</p>
+                </li>
+                <li class="program">
+                    <i class="fa-solid fa-store icon"></i>
+                    <h3 class="content-list">Book Exhibition</h3>
+                    <p class="description">lorem ipsum  dnnihdis ushoahaihs auhufs aufhuuav urhaijrjajsdkjf 
+                        a aprheh saivia u wiainuvbuhr s olbuhids urola urhu ubrbu aouh vuauhruh;
+                        ajpaonplav ovbola;lvo rbvyoou fuvbr afubaou huahru ar vhaluh fuahropairuoi 
+                        avuwbdbuanuoawborlandubawoub dsauvuabwvboaubv uabvuibwyrguaiih 
+                        auwosurhao ousfafgu  ybbiy hbbiviyv kbjo oiub dob oabofy bcakbf asv
+                        bbasbalkslbvosuab fkb sajbdvusufukjasdb fb asdb</p>
+                </li>
+                <li class="program">
+                    <i class="fa-solid fa-comment icon"></i>
+                    <h3 class="content-list">Book Forum</h3>
+                    <p class="description">lorem ipsum  dnnihdis ushoahaihs
+                         auhufs aufhuuav urhaijrjajsdkjf a aprheh saivia u wiainuvb
+                         uhr s olbuhids urola urhu ubrbu aouh vuauhruh;ajpaonplav ovbola;
+                         lvo rbvyoou fuvbr afubaou huahru ar vhaluh fuahropairuoi avuwbdbuasourwual
+                         dsnuoaoub dsauvuabwvboaubv uabvuibwyrguaiih auwosurhao ousfafgu  ybb
+                         iy hbbiviyv kbjo oiub dob oabofy bcakbf asvbbasbalkslbvosuab fkb sajbdvusufukjasdb fb asdb</p>
+                </li>
+                <li class="program">
+                  <i class="fa-sharp fa-solid fa-book-open-reader"></i>
+                    <h3 class="content-list">Literature Review</h3>
+                    <p class="description">lorem ipsum  dnnihdis ushoahaihs auhufs aufhuuav urhaijrjajsdkjf a 
+                        aprheh saivia u wiainuvbuhr s olbuhids urola urhu ubrbu aouh vuauhruh;ajpaonplav ovbola;
+                        lvo rbvyoou fuvbr afubaou huahru ar vhaluh fuahropairuoi avuwbdbuasourwualdsnuoawbo
+                        rlandubawoub dsauvuabwvboaubv uabvuibwyrguaiih auwosurhao ousfafgu  ybbiy hbbiviyv kbj
+                        o oiub dob oabofy bcakbf asvbbasbalkslbvosuab fkb sajbdvusufukjasdb fb asdb</p>
+                </li>
+            </ul>
+            <a href="#" class="details">SEE FURTHER DETAILS</a>
+            <button class="btn" id="joinBtn">Join BOOK FESTIVAL 2023</button>
+        </div>
+      </section>

```

Comment:
- [ ] It would be better if you stick to the layout of the given design as specified in the project requirements. Kindly consider adding one more card to the Main program section as your design has only 4 instead of 5.
- [ ] Also your design colors design match the given design. Please make sure your design color matches the given design as it's a requirement and needs to be implemented in other for your project to be approved. See below for the project requirement details:
<img width="1088" alt="Screenshot 2023-06-01 at 11 15 26" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/49dbdc4c-cc6a-4066-80b3-b3752872afdf">

<table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="985" alt="Screenshot 2023-06-01 at 10 56 16" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/38fd88f3-34a0-4ee7-899d-a7a309f7d9b0">
</td>
<td>
<img width="1440" alt="Screenshot 2023-06-01 at 10 55 16" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/007cb137-1a3a-43a9-aae8-2385b94394d3">
</td>
</tr>
</table>

- [ ] Kindly make sure that your website is fully responsive across all screen sizes. This will help create a good user experience. Your main program section cards are not responsive as expected. It would be better if you take a second look at it to fix this issue. This issue is between `768px` to `1200px`. See below for more details:
<img width="911" alt="Screenshot 2023-06-01 at 11 21 51" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/c440766c-fe5b-405c-9197-81df42587457">